build.gradle.kts

```diff
@@ -13,6 +13,7 @@ repositories {
 
 dependencies {
     testImplementation(kotlin("test"))
+    testImplementation("io.mockk:mockk:1.13.5")
```

Comment:
Please remove those two new dependencies (`mockk` and `logback`), we don't need them. It's very important to *avoid* adding dependencies unless they are really required, keep it in mind for later, dependencies introduce a maintenance cost and potential security issues!